src/pds/registrysweepers/driver.py

```diff
@@ -107,7 +107,12 @@ def run_factory(sweeper_f: Callable) -> Callable:
     args = parser.parse_args()
 
     # Define default sweepers to be run here, in order of execution
-    sweepers = [repairkit.run, provenance.run, ancestry.run, reindexer.run]
+    sweepers = [
+        repairkit.run,
+        provenance.run,
+        # ancestry.run,
```

Comment:
My pet peeve: commented-out code that doesn't explain why it's commented-out 😉

Reply:
Branch was reset to pre-lint state, changes were re-reviewed to ensure no unintended changes would be committed, and branch has been updated/pushed.

Awaiting branch-test results.